views/compliance-checker.jade

```diff
@@ -0,0 +1,11 @@
+extends layout
+
+block body
+  div#main-container.container.compliance-checker
+    div.row
+      div.col-md-12
+        h2 Compliance Checker 
+        p Here's your chance to check it out and provide feedback! <br/> <a href='http://data.ioos.us/compliance/index.html'>http://data.ioos.us/compliance/index.html</a>
```

Comment:
Can you point to https://compliance.ioos.us/index.html instead?

Reply:
this is all set and pushed to the branch.